### ISSUE TITLE
Search History not saving anymore

### ISSUE THREAD

**Issue by GHOSTJR12** [DestinyItemManager/DIM#8671]
### DIM Version

Version 7.31.1.1001940 and 7.31.1 normal version 

### Browser Details

Chrome 104.0.5112.101

### OS Details

both Mac Monterey 12.5.1 and Windows 10 fully updtad 

### Describe the bug

Just recently, all my Search Favoriats and History are now missng and will no longer save,   I have created new Favoriets Saves but when I close the app and open agian  all seaches are gone..     I have attempted to clear Chrome cookies and reset chrome and also tried another CPU (iMac) and same issue.  this happens on Beta and GM How can I fix this?

### Logs

```shell
[app] DIM v7.31.1.1001940 (beta) - Please report any errors to https://www.github.com/DestinyItemManager/DIM/issues
instrument.ts:123 Language Detected: en-us
instrument.ts:123 Language Assigned: en
instrument.ts:123 [dim sync] globalSettings Object
log.ts:58 Load manifest: 2808.10302734375 ms
4611686018430432072/d2/inventory:1 
        
       Access to fetch at 'https://www.bungie.net/D1/Platform/Destiny/2/Account/4611686018429124598/' from origin 'https://beta.destinyitemmanager.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
www.bungie.net/D1/Platform/Destiny/2/Account/4611686018429124598/:1 
        
      
        
        
      
        
      
       
        
       Failed to load resource: net::ERR_FAILED
instrument.ts:123 
        
       [accounts] Error getting D1 characters for Object DimError: You may not be connected to the internet, or an ad blocking or privacy extension may be blocking Bungie.net.
    at bungie-service-helper.ts:117:11
    at bungie-service-helper.ts:92:7
    at async m (destiny1-api.ts:24:9)
    at async k (destiny-account.ts:189:11)
    at async Promise.all (index 2)
    at async destiny-account.ts:184:10
    at async destiny-account.ts:88:13
    at async platforms.ts:97:13
(anonymous) @ instrument.ts:123
4611686018430432072/d2/inventory:1 
        
       Access to fetch at 'https://www.bungie.net/D1/Platform/Destiny/1/Account/4611686018430432072/' from origin 'https://beta.destinyitemmanager.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
www.bungie.net/D1/Platform/Destiny/1/Account/4611686018430432072/:1 
        
      
        
        
      
        
      
       
        
       Failed to load resource: net::ERR_FAILED
instrument.ts:123 
        
       [accounts] Error getting D1 characters for Object DimError: You may not be connected to the internet, or an ad blocking or privacy extension may be blocking Bungie.net.
    at bungie-service-helper.ts:117:11
    at bungie-service-helper.ts:92:7
    at async m (destiny1-api.ts:24:9)
    at async k (destiny-account.ts:189:11)
    at async Promise.all (index 1)
    at async destiny-account.ts:184:10
    at async destiny-account.ts:88:13
    at async platforms.ts:97:13
(anonymous) @ instrument.ts:123
instrument.ts:123 [dim sync] Saving profile data to IDB
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
instrument.ts:123 
        
       [d2-stores] Missing Item Definition:

 Object 

This item is not in the current manifest and will be added at a later time by Bungie.
(anonymous) @ instrument.ts:123
log.ts:58 Process inventory: 267.26611328125 ms
log.ts:58 Inventory state update: 32.22802734375 ms
4611686018430432072/d2/inventory:1 [Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://go.microsoft.com/fwlink/?linkid=2048113
instrument.ts:123 [dim sync] Saving profile data to IDB
```

**Comment by delphiactual** [comment]
Dupe #8665
